Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -430,7 +430,7 @@ private PluginInfo[] LoadPlugins(Plugin pluginSettings)
                 _logger.Warn(
                     $"To configure plugins update the '{pluginFolderSettingsName}' setting with either an absolute path, a path relative to the 'Ed-Fi-ODS-Implementation/Application/EdFi.Ods.WebApi/', or a path relative to the deployed EdFi.Ods.WebApi executable.");
 
-                return new PluginInfo[0];
+                return Array.Empty<PluginInfo>();
```

Comment:
In this PR, many array instantiations have been identified and changed to use `Array.Empty<T>`, which results in no new allocations to the heap that the GC has to clean up.